File: engine/engine-model/src/main/scala/tools/packet/mtsoperation/GetNotYetSoldMTSInventory.scala
package tools.packet.mtsoperation

import net.opcodes.SendOpcode
import server.MTSItemInfo
import tools.packet.PacketInput

class GetNotYetSoldMTSInventory(private var _items: java.util.List[MTSItemInfo]) extends PacketInput {
  def items: java.util.List[MTSItemInfo] = _items

  override def opcode(): SendOpcode = SendOpcode.MTS_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/mtsoperation/GetNotYetSoldMTSInventory.java
package tools.packet.mtsoperation;

import java.util.List;

import net.opcodes.SendOpcode;
import server.MTSItemInfo;
import tools.packet.PacketInput;

public record GetNotYetSoldMTSInventory(List<MTSItemInfo> items) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.MTS_OPERATION;
   }
}